script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -58,7 +58,7 @@ For a while after I woke up, I couldn't move from the bed.
 Looking up, I could see the sky outside, lofty as ever.
 The curtains fluttered and swayed.
 A dry breeze stroked my cheek.
-The stainless, immaculate blue made my eyes spin.
+The stainless, immaculate blue made my eyes spin. //Head spin 目のなかをくらくらさせる。
```

Comment:
Dazzled my eyes?

Reply:
Make "" eyes dazzle?

Reply:
I think "dazzled" my eyes is kinda strange. I like "made my head spin" even if it's liberal.